aot-std-optimizers/src/test/groovy/io/micronaut/aot/std/sourcegen/YamlPropertySourceGeneratorTest.groovy

```diff
@@ -47,9 +51,45 @@ public class Test_configStaticPropertySource extends MapPropertySource {
         put("micronaut.application.name", "demoApp");
         put("micronaut.server.port", 8181);
         put("micronaut.server.cors.enabled", true);
+        put("micronaut.security.intercept-url-map", list0());
         }});
   }
 
+  private static List list2() {
+    List result = new ArrayList<>();
```

Comment:
can these arrays be sized to the size of the data?

Reply:
done